apps/labs/posts/intro-to-pytorch-ignite.md

```diff
@@ -0,0 +1,1418 @@
+
+PyTorch-Ignite takes a \"Do-It-Yourself\" approach as research is
+unpredictable and it is important to capture its requirements without
+blocking things.
+
+## 🔥 PyTorch + Ignite 🔥
+
+PyTorch-Ignite wraps native PyTorch abstractions such as Modules,
+Optimizers, and DataLoaders in thin abstractions which allow your models
+to be separated from their training framework completely. This is
+achieved by a way of inverting control using an abstraction known as the
+**Engine**. The **Engine** is responsible for running an arbitrary
+function - typically a training or evaluation function - and emitting
+events along the way.
+
+A built-in event system represented by the **Events** class ensures
+`Engine`\'s flexibility, thus facilitating interaction on each step of
+the run. With this approach users can completely customize the flow of
+events during the run.
+
+In summary, PyTorch-Ignite is
+
+-   Extremely simple engine and event system = Training loop abstraction
+-   Out-of-the-box metrics to easily evaluate models
+-   Built-in handlers to compose training pipelines, save artifacts and
+    log parameters and metrics
+
+Additional benefits of using PyTorch-Ignite are
+
+-   Less code than pure PyTorch while ensuring maximum control and
+    simplicity
+-   More modular code
+
+```
+<table>
+<tr>
+
+<th>
+PyTorch-Ignite
+</th>
+
+<th>
+</th>    
+
+<th>
+</th>    
+
+<th>
+</th>    
+    
+<th>
+</th>    
+
+<th>
+</th>    
+
+<th>
+</th>    
+
+<th>
+</th>    
+
+    
+<th>
+Pure PyTorch
+</th>
+        
+</tr>
+        
+</table>
+```
+`<a href="https://raw.githubusercontent.com/pytorch/ignite/master/assets/ignite_vs_bare_pytorch.png">``<img width=800  src="https://raw.githubusercontent.com/pytorch/ignite/master/assets/ignite_vs_bare_pytorch.png"/>``</a>`
+
+### About the design of PyTorch-Ignite
+
+PyTorch-Ignite allows you to compose your application without being
+focused on a super multi-purpose object, but rather on weakly coupled
+components allowing advanced customization.
+
+The design of the library is guided by:
+
+-   Anticipating new software or use-cases to come in in the future
+    without centralizing everything in a single class.
+-   Avoiding configurations with a ton of parameters that are
+    complicated to manage and maintain.
+-   Providing tools targeted to maximizing cohesion and minimizing
+    coupling.
+-   Keeping it simple.
+
+# Quick-start example
+
+In this section we will use PyTorch-Ignite to build and train a
+classifier of the well-known [MNIST](http://yann.lecun.com/exdb/mnist/)
+dataset. This simple example will introduce the principal concepts
+behind PyTorch-Ignite.
+
+For additional information and details about the API, please, refer to
+the project\'s [documentation](https://pytorch.org/ignite/).
+
+``` python
+!pip install pytorch-ignite
+```
+
+    Collecting pytorch-ignite
+    ent already satisfied: torch<2,>=1.3 in /usr/local/lib/python3.6/dist-packages (from pytorch-ignite) (1.6.0+cu101)
```

Comment:
I'm getting the Vercel build error: `error [Error: [next-mdx-remote] error compiling MDX:`

Full error message:
```
Unexpected character `'` (U+0027) after name,
expected a character that can start an attribute name,
such as a letter, `$`, or `_`; whitespace before attributes;
or the end of the tag

  152 |
  153 |     Collecting pytorch-ignite
> 154 |     ent already satisfied: torch<2,>=1.3 in /usr/local/lib/python3.6/dist-packages (from pytorch-ignite) (1.6.0+cu101)
      |                                  ^
  155 |     Requirement already satisfied: future in /usr/local/lib/python3.6/dist-packages (from torch<2,>=1.3->pytorch-ignite) (0.16.0)
  156 |     Requirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (from torch<2,>=1.3->pytorch-ignite) (1.18.5)
```

[Link to error](https://vercel.com/quansight/labs/2Ww54VqR536ArhELPPukMnvANbm4#:~:text=Unexpected%20character)

The context is showing the result of some code. [Link to this section in the old blog website.](https://labs.quansight.org/blog/2020/09/pytorch-ignite#:~:text=0%3A00%3A01-,Requirement%20already%20satisfied,-%3A%20torch%3C2%2C%3E%3D1.3)

Similar problem to #647: output of code blocks formatted as plain text, not in code blocks, causing problems to Vercel